.travis.yml

```diff
@@ -14,24 +14,147 @@ git:
 
 sudo: false
 
+cache:
+  ccache: true
+  apt: true
+  directories:
+    - "${HOME}/mpich"
+    - "${HOME}/.ccache"
```

Comment:
Trying to use ccache to speed up the build. So far it works for mpich, but not tau